Src/Components/TeamMemberComponent.js

```diff
@@ -0,0 +1,29 @@
+
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import CardComponent from "./CardComponent";
+
+const TeamMemberComponent = () => {
+    const [memberInfo,setMemberInfo] = useState([]);
+    const { id } = useParams();
+    
+
+    useEffect(()=> {
+        fetchTeamMemberInfo();
+    },[]);
+
+    async function fetchTeamMemberInfo () {
+        const data = await fetch ("https://api.github.com/users/"+id );
+        const json = await data.json();
```

Comment:
Name this something else like data etc. json doesn't sound like the right variable name.